README.md

```diff
@@ -1,38 +1,31 @@
-# create-svelte
+# appbuilder-pwa
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+A template for building a Progressive Web App with Scripture App Builder.
 
-## Creating a project
+## Usage
 
-If you're seeing this, you've probably already done this step. Congrats!
+You will need to have Scripture App Builder 10.0.1 or newer to use this project without the provided example data.
 
-```bash
-# create a new project in the current directory
-npm init svelte
+### Develop
 
-# create a new project in my-app
-npm init svelte my-app
-```
+Once you've installed dependencies with `npm install`, you will need to convert data from one of the data folders before you can start a development server.  
+This can be accomplished in two ways:
 
-## Developing
+-   Run `npm run convertExamples` in the console to use the example data folder.
+-   Run `Build PWA Data Files` in Scripture App Builder.
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+If you are using the example data, run `npx svelte-kit dev` to start a development server. Otherwise, `npm run dev` will automatically convert the files in the folder copied to by `Build PWA Data Files`.
```

Comment:
Add something like "Contact chris_hubbard@sil.org for the feature key to enable `Build PWA Data Files`".